source/installation/kafka-deployments/install-docker.txt

```diff
@@ -15,3 +15,77 @@ Install with a Containerized Cluster Using Docker
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka in a containerized environment 
+using `Docker <https://docs.docker.com/engine/>`__. This method configures Kafka 
+to store data locally in Docker containers. Use this method if you want to learn 
+how to configure your own multi-server Kafka environment. 
+
+.. warning::
+
+   This deployment method is not recommended for production workloads. 
+
+About this Task
+---------------
+
+`Docker volumes <https://docs.docker.com/storage/volumes/>`__ can be used for 
+persisting data if a container stops running or is restarted. 
+
+Before you Begin
+----------------
+
+- You must have Docker installed on your computer. For more information, see  
+  `Install Docker Engine <https://docs.docker.com/engine/install/>`__. 
+
+- Download the following example ``docker-compose`` that configures a Kafka 
```

Comment:
We don't need to say "following example", since the file will just come from the download center. We are still waiting on this to be updated but we can link to https://www.mongodb.com/try/download/relational-migrator and ask the user to select "Docker" as a platform and then choose "Kafka reference implementation" as the file.

Reply:
Updated!